Tools/TestWebKitAPI/Tests/WebKitCocoa/ResourceLoadStatistics.mm

```diff
@@ -1415,3 +1415,40 @@ - (void)webView:(WKWebView *)webView stopURLSchemeTask:(id <WKURLSchemeTask>)tas
     }];
     TestWebKitAPI::Util::run(&done);
 }
+
+TEST(ResourceLoadStatistics, UserAgentStringForSite)
+{
+    auto configuration = adoptNS([[WKWebViewConfiguration alloc] init]);
+    auto dataStore = adoptNS([[WKWebsiteDataStore alloc] _initWithConfiguration:adoptNS([[_WKWebsiteDataStoreConfiguration alloc] init]).get()]);
+    [configuration setWebsiteDataStore:dataStore.get()];
+
+    auto webView = adoptNS([[TestWKWebView alloc] initWithFrame:NSMakeRect(0, 0, 100, 100) configuration:configuration.get()]);
```

Comment:
This seems like the default web view configuration; would it work if you just used `-initWithFrame:`?

```suggestion
    auto webView = adoptNS([[TestWKWebView alloc] initWithFrame:NSMakeRect(0, 0, 100, 100)]);
```

(And then replace `dataStore` below with `[webView configuration].dataStore`)